qa/workunits/cephtool/test.sh

```diff
@@ -2854,6 +2859,7 @@ OSD_TESTS+=" tiering_agent"
 OSD_TESTS+=" admin_heap_profiler"
 OSD_TESTS+=" osd_tell_help_command"
 OSD_TESTS+=" osd_compact"
+OSD_TESTS+=" osd_daemon_histogram"
```

Comment:
```
2021-07-29T08:48:54.174 INFO:tasks.workunit.client.0.smithi143.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:2960: main:  test_osd_daemon_histogram
2021-07-29T08:48:54.174 INFO:tasks.workunit.client.0.smithi143.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:2731: test_osd_daemon_histogram:  sudo ceph daemonhistogram osd.all osd.op_r_latency_out_bytes_histogram batch 1 3
2021-07-29T08:48:54.654 INFO:tasks.workunit.client.0.smithi143.stderr:no valid command found; 10 closest matches:
2021-07-29T08:48:54.654 INFO:tasks.workunit.client.0.smithi143.stderr:pg stat
2021-07-29T08:48:54.655 INFO:tasks.workunit.client.0.smithi143.stderr:pg getmap
2021-07-29T08:48:54.655 INFO:tasks.workunit.client.0.smithi143.stderr:pg dump [<dumpcontents:all|summary|sum|delta|pools|osds|pgs|pgs_brief>...]
2021-07-29T08:48:54.655 INFO:tasks.workunit.client.0.smithi143.stderr:pg dump_json [<dumpcontents:all|summary|sum|pools|osds|pgs>...]
2021-07-29T08:48:54.656 INFO:tasks.workunit.client.0.smithi143.stderr:pg dump_pools_json
2021-07-29T08:48:54.656 INFO:tasks.workunit.client.0.smithi143.stderr:pg ls-by-pool <poolstr> [<states>...]
2021-07-29T08:48:54.656 INFO:tasks.workunit.client.0.smithi143.stderr:pg ls-by-primary <id|osd.id> [<pool:int>] [<states>...]
2021-07-29T08:48:54.656 INFO:tasks.workunit.client.0.smithi143.stderr:pg ls-by-osd <id|osd.id> [<pool:int>] [<states>...]
2021-07-29T08:48:54.656 INFO:tasks.workunit.client.0.smithi143.stderr:pg ls [<pool:int>] [<states>...]
2021-07-29T08:48:54.657 INFO:tasks.workunit.client.0.smithi143.stderr:pg dump_stuck [<stuckops:inactive|unclean|stale|undersized|degraded>...] [<threshold:int>]
2021-07-29T08:48:54.657 INFO:tasks.workunit.client.0.smithi143.stderr:Error EINVAL: invalid command
```

see https://pulpito.ceph.com/kchai-2021-07-29_08:05:42-rados-wip-kefu-testing-2021-07-29-1336-distro-basic-smithi/6300873/

Reply:
@tchaikov - thanks for testing! This however looks like a broken/uncomplete ceph tool deployment - the daemonhistogram call from the test case looks valid but not available in ceph tool. I presume legacy tool version is in use. I added a ceph tool help call prior to daemonhistogram invocation to verify that proper tool is in use. Could you please re-run the QA.